include/freerdp/input.h

```diff
@@ -105,6 +109,8 @@ extern "C"
 	                                                           UINT16 code);
 	FREERDP_API BOOL freerdp_input_send_mouse_event(rdpInput* input, UINT16 flags, UINT16 x,
 	                                                UINT16 y);
+	FREERDP_API BOOL freerdp_input_send_rel_mouse_event(rdpInput* input, UINT16 flags, INT16 x,
```

Comment:
Perhaps this PR should also have the function for sending QoEEvent packets...

Reply:
added